app/components/file_upload_preview/file_upload_item/file_upload_item.js

```diff
@@ -118,6 +118,7 @@ export default class FileUploadItem extends PureComponent {
 
         const headers = {
             Authorization: `Bearer ${Client4.getToken()}`,
```

Comment:
Is there some reason why we need to set the `Authorization` header here as well? If the cookie is being sent, the `Authorization` header is redundant.

Reply:
I can try removing the authorization header, but We explicitly added it cause it was failing authentication before, dunno if something in the network layer changed for that